opencontrail/files/2.2/contrail-tor-agent.conf

```diff
@@ -13,7 +12,7 @@
 # server=10.0.0.1 10.0.0.2
 
 [DEFAULT]
-agent_name={{ system.name }}-{{ number }}
+agent_name={{ pillar.linux.system.name }}-{{ number }}
```

Comment:
this is fix for CI to pass, system.name is populated by reclass service which is not available if you don't use reclass, but pure pillars.